auslib/test/admin/views/test_history.py

```diff
@@ -96,6 +96,41 @@ def testFieldViewRelease(self):
         self.assertTrue('"fakePartials": true' in ret.data)
         self.assertTrue('"fakePartials": false' in ret.data)
 
+    def testFieldViewDiffFirstRelease(self):
+        # Add first release
+        blob = """
+        {
+            "name": "ddd1",
+            "schema_version": 1,
+            "detailsUrl": "blah",
+            "fakePartials": true,
+            "hashFunction": "sha512",
+            "platforms": {
+                "p": {
+                    "locales": {
+                        "dd": {
+                            "complete": {
+                                "filesize": 1234,
+                                "from": "*",
+                                "hashValue": "abc"
+                            }
+                        }
+                    }
+                }
+            }
+        }"""
+
+        ret = self._put('/releases/ddd1', data=dict(blob=blob, name='ddd1',
+                        product='d', data_version=1))
+        self.assertStatusCode(ret, 201)
+        table = dbo.releases.history
+        row, = table.select(order_by=[table.change_id.desc()], limit=1)
```

Comment:
I think you want .asc() here. Right now, this ends up selecting the last change, not the first one. r=me with that fixed.